Civi/Funding/Api4/Action/FundingCaseInfo/GetFieldsAction.php

```diff
@@ -222,6 +222,7 @@ public function getRecords(): array {
         'title' => 'application_process_status',
         'data_type' => 'String',
         'options' => $this->getOptions(FundingApplicationProcess::getEntityName(), 'status'),
+        'operators' => ['IN'],
```

Comment:
Just doing this works as long as we do not need pagination. Thus, the condition should be part of the SQL statement. The code at https://github.com/systopia/funding/blob/726567d69962f74dc07a60b9120400cb898722d0/Civi/Funding/Api4/Action/FundingCaseInfo/GetAction.php#L174 could be a adopted accordingly. (Currently this is also an issue for funding program title.)